src/client.rs

```diff
@@ -2717,6 +2752,43 @@ mod test {
         assert_eq!(manifest.config.digest, pulled_manifest.config.digest);
     }
 
+    #[tokio::test]
+    #[cfg(feature = "test-registry")]
+    async fn test_raw_manifest_digest() {
```

Comment:
This test is not making any request against the local registry. You can drop the feature `test-restry` and the code that deal with starting the local registry

Reply:
Ah correct! In a previous iteration I did have it use the local registry, but it didn't really add any assurances to the test.